style.css

```diff
@@ -0,0 +1,58 @@
+html {
+  height: 100%;
+}
+
+body {
+  background: rgb(182, 21, 156);
+  background: linear-gradient(0deg, rgba(182, 21, 156, 1) 20%, rgba(7, 40, 176, 1) 100%);
+  height: 100%;
+}
+
+#main {
+  height: 100%;
+}
+
+#title {
+  display: flex;
+  justify-content: center;
+  font-family: 'Teko', sans-serif;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+  height: auto;
+  margin: 0 auto;
+}
+
+#img-caption {
+  display: flex;
+  justify-content: center;
+  font-family: 'Oswald', sans-serif;
+  color: yellow;
+}
+
+#tribute-info {
+  display: flex;
+  justify-content: center;
+  font-family: 'Oswald', sans-serif;
+  color: antiquewhite;
+}
```

Comment:
Please, try to group common code using classes instead of writing the same code multiple times, that makes the code easier to read and help you become a better coder 🤓 👍